dashboard/config/locales/en.yml

```diff
@@ -620,11 +620,10 @@ en:
   compatibility:
     upgrade:
       unsupported_message: "Your browser is not supported. Please upgrade your browser to [one of our supported browsers](%{supported_browsers_url}). You can try viewing the page, but expect functionality to be broken."
+      ie_unsupported_message: "We no longer support this browser. Please try visiting this page using Edge, Chrome, Firefox, or another modern browser."
```

Comment:
Why have the new `ie_unsupported_message` if the `unsupported_message` string already exists?

Reply:
The copy change was listed under the [deliverable section in the ticket](https://codedotorg.atlassian.net/browse/FND-1615). Assumed the existing message should continue to exist for all other browsers that just need to be upgraded.